src/containers/shared/components/TransactionTable/TransactionTableRow.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { Link } from 'react-router-dom'
+import TxLabel from '../TxLabel'
+import { ReactComponent as SuccessIcon } from '../../images/success.svg'
+import { ReactComponent as FailIcon } from '../../images/ic_fail.svg'
+import TxDetails from '../TxDetails'
+import { localizeDate } from '../../utils'
+import './styles.scss'
+import { useLanguage } from '../../hooks'
+
+const TIME_ZONE = 'UTC'
```

Comment:
should we have a global timezone setting maybe?

Reply:
i think there is one inside `transacactionUtils.ts` that renders date in non-numeric format (eg, "December 20, 2020 at 3:23:16 AM").  But I also see many modules define their own date-options for numeric dates (eg, "12/20/2020, 3:23:16 AM"), which indeed have duplicated code. So I think it should be part of another issue if we want to clean up and rename the date options. @ckniffen correct me if im wrong

Reply:
Let make it part of something else